tests/testdata/keras_tuner_cifar_example.py

```diff
@@ -0,0 +1,112 @@
+import argparse
+import os
+
+from tensorflow import keras
+from tensorflow.keras import layers
+from tensorflow.keras.datasets import cifar10
+
+from tensorflow.keras.preprocessing.image import ImageDataGenerator
+
+from kerastuner.tuners import RandomSearch
+from kerastuner.engine.hypermodel import HyperModel
+from kerastuner.engine.hyperparameters import HyperParameters
+
+(x_train, y_train), (x_test, y_test) = cifar10.load_data()
+y_train = keras.utils.to_categorical(y_train, 10)
+y_test = keras.utils.to_categorical(y_test, 10)
+x_train = x_train.astype("float32")
+x_test = x_test.astype("float32")
+x_train /= 255
+x_test /= 255
+
+
+def build_model(hp):
+    model = keras.Sequential()
```

Comment:
Comments about the architecture:

- No need for dropout after a Conv, only use dropout before a Dense
- To justify the use of 4 V100, we should do architecture search, not just search for the learning rate and dropout rates

I recommend the follow search:

```
activation = choice[ReLU, LeakyReLU, ELU]
for i in range(int[2-4]):  # blocks
  for j in range(int[1-3]):  # one block's content
    conv(int[16-256], same)
    activation
  maxpooling(2)  # end of block
dropout(float[0-1])
dense
```

Reply:
Done.

Reply:
Used just 'relu' and 'elu' activations as hp.Choice does not allow objects of different types. I could not use layers.Activation('relu') and layers.LeakyRelu() together in hp.Choice.